ravenframework/Models/EnsembleModel.py

```diff
@@ -471,9 +471,14 @@ def collectOutput(self,finishedJob,output):
     joinedResponse['prefix'] = np.asarray([finishedJob.identifier])
     if joinedIndexMap:
       joinedResponse['_indexMap'] = np.atleast_1d(joinedIndexMap)
+
     if output.name not in optionalOutputNames:
       if output.name not in targetEvaluationNames.keys():
-        output.addRealization(joinedResponse)
+        if isinstance(evaluation,dict):
```

Comment:
Could you update this line with followings:
```Python
    # in the event a batch is run, the evaluations will be a dict as {'RAVEN_isBatch':True, 'realizations': [...]}
    if isinstance(evaluation,dict) and evaluation.get('RAVEN_isBatch',False):
```

Reply:
added